daq/host.py

```diff
@@ -271,6 +277,22 @@ def _state_transition(self, target, expected=None):
         LOGGER.debug('Target port %d state: %s -> %s', self.target_port, self.state, target)
         self.state = target
 
+    def _build_switch_info(self) -> usi.SwitchInfo:
+        switch_config = self._get_switch_config()
+        switch_model = switch_config["model"]
+        if switch_model in usi.SwitchModel.keys():
+            switch_model = usi.SwitchModel.Value(switch_model)
+        else:
+            switch_model = usi.SwitchModel.OVS_SWITCH
```

Comment:
seems like this should be a hard error not default.  What's the logic behind defaulting to OVS_SWITCH with when switch model is, e.g.,  ALIED_TELESIS_X230?